packages/flutter_tools/test/integration/flutter_tester_test.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/build_info.dart';
+import 'package:flutter_tools/src/device.dart';
+import 'package:flutter_tools/src/tester/flutter_tester.dart';
+import 'package:test/test.dart';
+
+import '../src/context.dart';
+
+void main() {
+  Directory tempDir;
+  Directory oldCurrentDir;
+
+  setUp(() async {
+    tempDir = await fs.systemTempDirectory.createTemp('flutter_tester_device');
+    oldCurrentDir = fs.currentDirectory;
+    fs.currentDirectory = tempDir;
+  });
+
+  tearDown(() {
+    fs.currentDirectory = oldCurrentDir;
+    try {
+      tempDir?.deleteSync(recursive: true);
```

Comment:
If we change the directory during the test, we'll need some way to indicate that only one test can run at a time (perhaps, at least using doc comments at the beginning of the library).

Reply:
I added `README.md` file, where I also explained that only one test can be run at a time.